src/utils/handlePollFallback.ts

```diff
@@ -0,0 +1,28 @@
+export const WAIT_TIME_BEFORE_START_POLLING = 20 * 1000
+export const POLLING_INTERVAL = 10 * 1000
```

Comment:
I'm of the opinion that these should be quite short - most files are generally small and 5s seems reasonable before we start the fallback

```suggestion
export const WAIT_TIME_BEFORE_START_POLLING = 5 * 1000
export const POLLING_INTERVAL = 1 * 1000
```